### PR TITLE
(backport to 1.4) Add additional IE11 support to load the App, Edge support to download console.vv files

### PR DESCRIPTION
**Pull request by sjd78**
Add support to load the web-ui app in IE11:
 - Use `babel-polyfil` to get the full set of polyfills required for IE11 (backport of #870)
 - Refactored the static index page processing and add meta tag to stay off compatibility mode (backport of #885)
 - Fix download of console.vv file (part of backport of #885, see #887)

**NOTE**: Due to the use of modern APIs and styles, some pages may not render perfectly.

### CODE REVIEW

File: config/webpack.config.dev.js [hunk 1]
```diff
@@ -196,39 +196,51 @@ module.exports = {
...
     // If you require a missing module and then `npm install` it, you still have
     // to restart the development server for Webpack to discover it. This plugin
     // makes the discovery automatic so you don't have to restart.
     // See https://github.com/facebookincubator/create-react-app/issues/186
     new WatchMissingNodeModulesPlugin(paths.appNodeModules),
 
     //global jquery is provided to any webpack modules
-    /*new webpack.ProvidePlugin({
+    /*
+    new webpack.ProvidePlugin({
```
Comment (bond95) [inline config/webpack.config.dev.js]: Is this still relevant, I think we can remove this comment.

File: package.json [hunk 2]
```diff
@@ -43,6 +43,7 @@
     "flow-babel-webpack-plugin": "1.1.1",
     "fs-extra": "0.30.0",
     "gzip-size": "3.0.0",
+    "handlebars-loader": "^1.7.0",
```
Comment (bond95) [inline package.json]: Remove, please, `^`

Comment (sjd78) [reply package.json]: Can't remove it.

It is needed for the template processing in the `HtmlWebpackPlugin` webpack plugin of the file `static/index.hbs`.  That is also why it is part of `devDependencies`.

File: config/webpack.config.dev.js (hunk 1 above)
Comment (sjd78) [reply config/webpack.config.dev.js]: Done.

File: package.json (hunk 2 above)
Comment (bond95) [reply package.json]: I mean:
`"handlebars-loader": "1.7.0",`

Comment (sjd78) [reply package.json]: Oh right.  Fixed the dependency version to just `"1.7.0"`.

**Comment by sjd78** [comment]
Fixed the rpm build in 1.4 by applying the commits from #896

**Comment by sjd78** [comment]
ci build please

**Comment by gregsheremeta** [comment]
new commit lgtm

**Comment by leistnerova** [comment]
ci build please

**Comment by leistnerova** [comment]
VM portal goes crazy with IE11. It changes information of VM randomly from previous values. 
No errors in console. See video.

[Screencast from 01-07-2019 06:09:38 PM.zip](https://github.com/oVirt/ovirt-web-ui/files/2733640/Screencast.from.01-07-2019.06.09.38.PM.zip)

Tested in IE 11.1 on windows 10
ovirt-web-ui-1.4.5-0.20190107git5da58c5.el7.noarch with ovirt-engine-4.2.8.1-0.1.el7ev.noarch

**Comment by sjd78** [comment]
@leistnerova, @gregsheremeta, @bond95, @lwrigh -- I vote for leaving the IE11 support on ovirt 4.2 (web-ui 1.4) as "good" as it is with this PR applied.  Any objections?

**Comment by bond95** [comment]
@sjd78 agree, it takes too much resources to make VM Portal work properly in IE11.

**Pull request merged**